pkg/csi/service/identity.go

```diff
@@ -66,13 +66,6 @@ func (s *service) GetPluginCapabilities(
 					},
 				},
 			},
-			{
-				Type: &csi.PluginCapability_VolumeExpansion_{
```

Comment:
Does removal of this work if we have a old k8s cluster? Which component has logic related to this capability? If its resizer then we are good, if kubelet is looking into this then we need to re-evaluate this code removal.

Reply:
@xing-yang had checked this and confirmed that this will not break older clusters.

Reply:
The VolumeExpansion plugin capability has never been checked by any Kubernetes components.  It will be removed in a future CSI spec release.  So it is safe to remove it.